presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -479,9 +479,12 @@ else if (!partitionColumnNames.contains(name)) {
         table.setPartitionKeys(partitionColumns);
         table.setSd(sd);
 
-        PrivilegeGrantInfo allPrivileges = new PrivilegeGrantInfo("all", 0, tableOwner, PrincipalType.USER, true);
+        PrivilegeGrantInfo selectPrivilege = new PrivilegeGrantInfo("select", 0, tableOwner, PrincipalType.USER, true);
```

Comment:
@tomz can you please explain the reason behind replacing "all" privileges with individual ones? The owner of the table gets all the privileges (with grant option) under Hive SQL standard authorization, so why is this needed?